src/common/Interfaces.ts

```diff
@@ -103,6 +103,7 @@ export abstract class Modal {
     abstract show(modal: React.ReactElement<Types.ViewProps>, modalId: string, options?: Types.ModalOptions): void;
     abstract dismiss(modalId: string): void;
     abstract dismissAll(): void;
+    abstract anyDisplayed(): boolean;
```

Comment:
Keeping with the naming conventions in ReactXP, this should be called "isAnyDisplayed". I'm not sure this functionality requires a new method. We could allow users to specify an empty string to "isDisplayed" if they want to query whether any modal is displayed.

Reply:
Also, any new exposed functionality should update the docs and the RXPTest sample.